library/elliptics.h

```diff
@@ -158,8 +158,9 @@ struct dnet_net_state
 	struct list_head	node_entry;
 	// To store at node::storage_state_list (List of all network-active states, used for unsheduling process)
 	struct list_head	storage_state_entry;
-	// To store list of all idc connected with this state
-	struct list_head	idc_list;
+	// Mapping backend_id -> struct dnet_idc
+	struct rb_root		idc_root;
+	pthread_rwlock_t        idc_lock;
```

Comment:
Please, add a description what this lock guards, because I see changes in idc_root which is made under read lock.